plugin/phpactor.vim

```diff
@@ -3,6 +3,8 @@ if exists('g:loaded_phpactor')
 endif
 
 let g:loaded_phpactor = 1
+
+" Config {{{
 let g:phpactorpath = expand('<sfile>:p:h') . '/..'
 let g:phpactorbinpath = g:phpactorpath. '/bin/phpactor'
```

Comment:
Unrelated - should probably deprecate this and create a documented `g:phpactorBinPath`